build/dependencies.yaml

```diff
@@ -166,7 +166,7 @@ dependencies:
     version: buster-v2.3.1
     refPaths:
     - path: build/common.sh
-      match: go_runner_version=
```

Comment:
@BenTheElder : Do we need one for setcap?

Reply:
if there's multiple references this is how we keep them in sync.